is not legal for a JDOM comment: Comments cannot contain double hyphens (--). -- migrating :enricher

**Version used**
MMA  1.3.0

**Describe the bug**
is not legal for a JDOM comment: Comments cannot contain double hyphens (--). -- migrating :enricher

***Expected Behavior***
Migrate enricher elemetns

***Actual Behavior***
Do nothing in cases of enricher

**Relevant logs**
Executing migrator 1.3.0...
WARN  2022-06-09 09:18:23,196 [main] com.mulesoft.tools.migration.step.ReportingStep: Exception org.jdom2.IllegalDataException: The data "<enricher xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
                    <dw:transform-message xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw" doc:name="CALENDARS-S-T-0004">
                        <dw:set-payload><![CDATA[%dw 1.0
%output application/java
---
{
	message:"Not able to retrieve calendars." as :string,
	status:"500" as :string,
	code:"CALENDARS-S-T-0004" as :string,
	origin: "database" as :string,
	causeMessage: "" as :string
}]]></dw:set-payload>
                    </dw:transform-message>
                </enricher>" is not legal for a JDOM comment: Comments cannot contain double hyphens (--). -- migrating :enricher
ERROR 2022-06-09 09:18:23,206 [main] com.mulesoft.tools.migration.engine.MigrationJob: Failed to apply task, rolling back and continuing with the next one.
com.mulesoft.tools.migration.exception.MigrationTaskException: Task execution exception. Task execution exception (The data "<enricher xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
                    <dw:transform-message xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw" doc:name="CALENDARS-S-T-0004">
                        <dw:set-payload><![CDATA[%dw 1.0
%output application/java
---
{
	message:"Not able to retrieve calendars." as :string,
	status:"500" as :string,
	code:"CALENDARS-S-T-0004" as :string,
	origin: "database" as :string,
	causeMessage: "" as :string
}]]></dw:set-payload>
                    </dw:transform-message>
                </enricher>" is not legal for a JDOM comment: Comments cannot contain double hyphens (--).) migrating node:
<enricher xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
                    <dw:transform-message xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw" doc:name="CALENDARS-S-T-0004">
                        <dw:set-payload><![CDATA[%dw 1.0
%output application/java
---
{
	message:"Not able to retrieve calendars." as :string,
	status:"500" as :string,
	code:"CALENDARS-S-T-0004" as :string,
	origin: "database" as :string,
	causeMessage: "" as :string
}]]></dw:set-payload>
                    </dw:transform-message>
                </enricher>
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.execute(AbstractMigrationTask.java:96) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.engine.MigrationJob.execute(MigrationJob.java:102) [mule-migration-tool-engine-1.3.0.jar:1.3.0]
	at com.mulesoft.tools.migration.MigrationRunner.run(MigrationRunner.java:82) [mule-migration-assistant-runner-1.3.0.jar:1.3.0]
	at com.mulesoft.tools.migration.MigrationRunner.main(MigrationRunner.java:67) [mule-migration-assistant-runner-1.3.0.jar:1.3.0]
Caused by: com.mulesoft.tools.migration.exception.MigrationStepException: Task execution exception (The data "<enricher xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
                    <dw:transform-message xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw" doc:name="CALENDARS-S-T-0004">
                        <dw:set-payload><![CDATA[%dw 1.0
%output application/java
---
{
	message:"Not able to retrieve calendars." as :string,
	status:"500" as :string,
	code:"CALENDARS-S-T-0004" as :string,
	origin: "database" as :string,
	causeMessage: "" as :string
}]]></dw:set-payload>
                    </dw:transform-message>
                </enricher>" is not legal for a JDOM comment: Comments cannot contain double hyphens (--).) migrating node:
<enricher xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
                    <dw:transform-message xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw" doc:name="CALENDARS-S-T-0004">
                        <dw:set-payload><![CDATA[%dw 1.0
%output application/java
---
{
	message:"Not able to retrieve calendars." as :string,
	status:"500" as :string,
	code:"CALENDARS-S-T-0004" as :string,
	origin: "database" as :string,
	causeMessage: "" as :string
}]]></dw:set-payload>
                    </dw:transform-message>
                </enricher>
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.lambda$fetchAndProcessNodes$8(AbstractMigrationTask.java:122) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask$$Lambda$90/1956746660.accept(Unknown Source) ~[?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) ~[?:1.8.0_51]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175) ~[?:1.8.0_51]
	at java.util.LinkedList$LLSpliterator.forEachRemaining(LinkedList.java:1235) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:502) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_51]
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418) ~[?:1.8.0_51]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.fetchAndProcessNodes(AbstractMigrationTask.java:116) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.lambda$execute$3(AbstractMigrationTask.java:75) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask$$Lambda$62/564086167.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1249) ~[?:1.8.0_51]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.execute(AbstractMigrationTask.java:73) ~[mule-migration-tool-api-1.3.0.jar:?]
	... 3 more
Caused by: org.jdom2.IllegalDataException: The data "<enricher xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
                    <dw:transform-message xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw" doc:name="CALENDARS-S-T-0004">
                        <dw:set-payload><![CDATA[%dw 1.0
%output application/java
---
{
	message:"Not able to retrieve calendars." as :string,
	status:"500" as :string,
	code:"CALENDARS-S-T-0004" as :string,
	origin: "database" as :string,
	causeMessage: "" as :string
}]]></dw:set-payload>
                    </dw:transform-message>
                </enricher>" is not legal for a JDOM comment: Comments cannot contain double hyphens (--).
	at org.jdom2.Comment.setText(Comment.java:124) ~[jdom2-2.0.6.jar:2.0.6]
	at org.jdom2.Comment.<init>(Comment.java:89) ~[jdom2-2.0.6.jar:2.0.6]
	at com.mulesoft.tools.migration.report.DefaultMigrationReport.report(DefaultMigrationReport.java:159) ~[mule-migration-tool-engine-1.3.0.jar:1.3.0]
	at com.mulesoft.tools.migration.report.DefaultMigrationReport.report(DefaultMigrationReport.java:121) ~[mule-migration-tool-engine-1.3.0.jar:1.3.0]
	at com.mulesoft.tools.migration.library.mule.steps.core.Enricher.execute(Enricher.java:120) ~[mule-migration-tool-library-1.3.0.jar:?]
	at com.mulesoft.tools.migration.library.mule.steps.core.Enricher.execute(Enricher.java:37) ~[mule-migration-tool-library-1.3.0.jar:?]
	at com.mulesoft.tools.migration.step.ReportingStep.execute(ReportingStep.java:55) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.step.ReportingStep.execute(ReportingStep.java:29) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.lambda$fetchAndProcessNodes$8(AbstractMigrationTask.java:119) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask$$Lambda$90/1956746660.accept(Unknown Source) ~[?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) ~[?:1.8.0_51]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175) ~[?:1.8.0_51]
	at java.util.LinkedList$LLSpliterator.forEachRemaining(LinkedList.java:1235) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:502) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_51]
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418) ~[?:1.8.0_51]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.fetchAndProcessNodes(AbstractMigrationTask.java:116) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.lambda$execute$3(AbstractMigrationTask.java:75) ~[mule-migration-tool-api-1.3.0.jar:?]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask$$Lambda$62/564086167.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1249) ~[?:1.8.0_51]
	at com.mulesoft.tools.migration.task.AbstractMigrationTask.execute(AbstractMigrationTask.java:73) ~[mule-migration-tool-api-1.3.0.jar:?]
	... 3 more
===============================================================================
MIGRATION ASSISTANT RUN SUCCESSFULLY
===============================================================================
Total time: 14,194 s

**Reproducible Test Case**

There are not double hyphens (--).  but it is not only in this case the beheivour is the same with enricher 

 <enricher target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
                    <dw:transform-message doc:name="CALENDARS-S-T-0004">
                        <dw:set-payload><![CDATA[%dw 1.0
%output application/java
---
{
	message:"Not able to retrieve calendars." as :string,
	status:"500" as :string,
	code:"CALENDARS-S-T-0004" as :string,
	origin: "database" as :string,
	causeMessage: "" as :string
}]]></dw:set-payload>
                    </dw:transform-message>
                </enricher>

--- a/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/input/src/main/app/mule-config.xml
+++ b/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/input/src/main/app/mule-config.xml
@@ -1,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:httpn="http://www.mulesoft.org/schema/mule/http"
+      xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+      xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd">
+
+    <httpn:listener-config name="httpConf" host="localhost" port="${httpPort}"/>
+    
+    <flow name="listenerContentType" >
+        <httpn:listener config-ref="httpConf" path="testInput"/>
+
+        <enricher target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
+            <dw:transform-message doc:name="CALENDARS-S-T-0004">
+                <dw:set-payload><![CDATA[%dw 1.0
+%output application/java
+---
+{
+	message:"Not able to retrieve calendars." as :string,
+	status:"500" as :string,
+	code:"CALENDARS-S-T-0004" as :string,
+	origin: "database" as :string,
+	causeMessage: "" as :string
+}]]></dw:set-payload>
+            </dw:transform-message>
+        </enricher>
+
+    </flow>
+    
+</mule>

--- a/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/output/pom.xml
+++ b/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/output/pom.xml
@@ -1,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.mule.migrated</groupId>
+  <artifactId>http1</artifactId>
+  <version>1.0.0-M4-SNAPSHOT</version>
+  <packaging>mule-application</packaging>
+  <description>Application migrated with MMA</description>
+  <dependencies>
+    <dependency>
+      <groupId>com.mulesoft.mule.modules</groupId>
+      <artifactId>mule-compatibility-module</artifactId>
+      <version>1.4.0</version>
+      <classifier>mule-plugin</classifier>
+    </dependency>
+    <dependency>
+      <groupId>org.mule.modules</groupId>
+      <artifactId>mule-scripting-module</artifactId>
+      <version>2.0.0</version>
+      <classifier>mule-plugin</classifier>
+    </dependency>
+    <dependency>
+      <groupId>org.mule.connectors</groupId>
+      <artifactId>mule-http-connector</artifactId>
+      <version>1.6.0</version>
+      <classifier>mule-plugin</classifier>
+    </dependency>
+  </dependencies>
+  <repositories>
+    <repository>
+      <id>mulesoft-releases</id>
+      <name>MuleSoft Releases Repository</name>
+      <url>https://repository.mulesoft.org/releases/</url>
+    </repository>
+    <repository>
+      <id>anypoint-exchange</id>
+      <name>Anypoint Exchange</name>
+      <url>https://maven.anypoint.mulesoft.com/api/v1/maven</url>
+    </repository>
+  </repositories>
+  <pluginRepositories>
+    <pluginRepository>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <id>mulesoft-releases</id>
+      <name>MuleSoft Releases Repository</name>
+      <url>https://repository.mulesoft.org/releases/</url>
+    </pluginRepository>
+  </pluginRepositories>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.mule.tools.maven</groupId>
+        <artifactId>mule-maven-plugin</artifactId>
+        <version>3.2.1</version>
+        <extensions>true</extensions>
+        <configuration />
+      </plugin>
+    </plugins>
+  </build>
+</project>

--- a/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/output/report/report.json
+++ b/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/output/report/report.json
@@ -1,0 +1,103 @@
+{
+  "projectType": "MULE_THREE_APPLICATION",
+  "projectName": "input",
+  "connectorsMigrated": [
+    "org.mule.connectors:mule-http-connector:1.6.0"
+  ],
+  "numberOfMuleComponents": 8,
+  "numberOfMuleComponentsMigrated": 7,
+  "componentDetails": {
+    "mule": {
+      "success": 1,
+      "failure": 0
+    },
+    "enricher": {
+      "success": 0,
+      "failure": 1
+    },
+    "flow": {
+      "success": 1,
+      "failure": 0
+    },
+    "sub-flow": {
+      "success": 1,
+      "failure": 0
+    },
+    "flow-ref": {
+      "success": 1,
+      "failure": 0
+    },
+    "ee:transform": {
+      "success": 1,
+      "failure": 0
+    },
+    "http:listener-config": {
+      "success": 1,
+      "failure": 0
+    },
+    "http:listener": {
+      "success": 1,
+      "failure": 0
+    }
+  },
+  "numberOfMELExpressions": 1,
+  "numberOfMELExpressionsMigrated": 1,
+  "numberOfMELExpressionLines": 1,
+  "numberOfMELExpressionLinesMigrated": 1,
+  "numberOfDWTransformations": 1,
+  "numberOfDWTransformationsMigrated": 1,
+  "numberOfDWTransformationLines": 10,
+  "numberOfDWTransformationLinesMigrated": 10,
+  "detailedMessages": [
+    {
+      "level": "WARN",
+      "key": "http.statusCode",
+      "component": "http:response",
+      "lineNumber": 10,
+      "columnNumber": 113,
+      "message": "Avoid using an outbound property to determine the status code.",
+      "filePath": "src/main/mule/mule-config.xml",
+      "documentationLinks": []
+    },
+    {
+      "level": "WARN",
+      "key": "http.statusCode",
+      "component": "http:error-response",
+      "lineNumber": 16,
+      "columnNumber": 141,
+      "message": "Avoid using an outbound property to determine the status code.",
+      "filePath": "src/main/mule/mule-config.xml",
+      "documentationLinks": []
+    },
+    {
+      "level": "WARN",
+      "key": "message.attributesToInboundProperties",
+      "component": "compatibility:attributes-to-inbound-properties",
+      "lineNumber": 24,
+      "columnNumber": 57,
+      "message": "Expressions that query \u0027inboundProperties\u0027 from the message should instead query the message \u0027attributes\u0027. Remove this component if there are no uses of \u0027inboundProperties\u0027 in expressions or components that rely on \u0027inboundProperties\u0027 (such as \u0027copy-properties\u0027).",
+      "filePath": "src/main/mule/mule-config.xml",
+      "documentationLinks": []
+    },
+    {
+      "level": "ERROR",
+      "key": "enricher.multipleEnrichments",
+      "component": "flow-ref",
+      "lineNumber": 24,
+      "columnNumber": 57,
+      "message": "Only the first enrichment pair is migrated. You should use a temporary structure to make multiple enrichments.",
+      "filePath": "src/main/mule/mule-config.xml",
+      "documentationLinks": []
+    },
+    {
+      "level": "WARN",
+      "key": "message.outboundProperties",
+      "component": "compatibility:outbound-properties-to-var",
+      "lineNumber": 53,
+      "columnNumber": 51,
+      "message": "Instead of using outbound properties in the flow, move the expression that sets the property into the XML attribute (such as \u0027method\u0027) of the operation or listener that accepts the expression.",
+      "filePath": "src/main/mule/mule-config.xml",
+      "documentationLinks": []
+    }
+  ]
+}

--- a/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/output/src/main/mule/mule-config.xml
+++ b/mule-migration-tool-e2e-tests/src/test/resources/e2e/enricher/http1/output/src/main/mule/mule-config.xml
@@ -1,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:compatibility="http://www.mulesoft.org/schema/mule/compatibility" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns:httpn="http://www.mulesoft.org/schema/mule/http" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd http://www.mulesoft.org/schema/mule/compatibility http://www.mulesoft.org/schema/mule/compatibility/current/mule-compatibility.xsd http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+
+    <http:listener-config name="httpConf">
+        <http:listener-connection host="localhost" port="${httpPort}" />
+    </http:listener-config>
+
+    <flow name="listenerContentType">
+        <http:listener config-ref="httpConf" path="testInput">
+            <http:response statusCode="#[migration::HttpListener::httpListenerResponseSuccessStatusCode(vars)]">
+                <!--Migration WARN: Avoid using an outbound property to determine the status code.-->
+                <!--    For more information refer to:-->
+                <!--        * https://github.com/mulesoft/mule-migration-assistant/blob/master/docs/user-docs/migration-tool-post-mig.adoc#outbound_properties-->
+                <http:headers>#[migration::HttpListener::httpListenerResponseHeaders(vars)]</http:headers>
+            </http:response>
+            <http:error-response statusCode="#[vars.statusCode default migration::HttpListener::httpListenerResponseErrorStatusCode(vars)]">
+                <!--Migration WARN: Avoid using an outbound property to determine the status code.-->
+                <!--    For more information refer to:-->
+                <!--        * https://github.com/mulesoft/mule-migration-assistant/blob/master/docs/user-docs/migration-tool-post-mig.adoc#outbound_properties-->
+                <http:headers>#[migration::HttpListener::httpListenerResponseHeaders(vars)]</http:headers>
+            </http:error-response>
+        </http:listener>
+
+        <compatibility:attributes-to-inbound-properties>
+            <!--Migration WARN: Expressions that query 'inboundProperties' from the message should instead query the message 'attributes'. Remove this component if there are no uses of 'inboundProperties' in expressions or components that rely on 'inboundProperties' (such as 'copy-properties').-->
+            <!--    For more information refer to:-->
+            <!--        * https://docs.mulesoft.com/mule-runtime/4.3/intro-mule-message#inbound-properties-are-now-attributes-->
+            <!--        * https://github.com/mulesoft/mule-migration-assistant/blob/master/docs/user-docs/migration-tool-post-mig.adoc#inbound_properties-->
+        </compatibility:attributes-to-inbound-properties>
+
+        <flow-ref name="listenerContentType_Enricher_0" target="sessionVars.exceptionMessage">
+            <!--Migration ERROR: Only the first enrichment pair is migrated. You should use a temporary structure to make multiple enrichments.-->
+            <!--<enricher xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" target="#[sessionVars.exceptionMessage]" doc:name="Exception message">
+            <dw:transform-message xmlns:dw="http://www.mulesoft.org/schema/mule/ee/dw" doc:name="CALENDARS-S-T-0004">
+                <dw:set-payload><![CDATA[%dw 1.0
+%output application/java
+ - - -
+{
+	message:"Not able to retrieve calendars." as :string,
+	status:"500" as :string,
+	code:"CALENDARS-S-T-0004" as :string,
+	origin: "database" as :string,
+	causeMessage: "" as :string
+}]]></dw:set-payload>
+            </dw:transform-message>
+        </enricher>-->
+        </flow-ref>
+
+        <compatibility:set-property propertyName="sessionVars.exceptionMessage" value="#[vars.sessionVars.exceptionMessage]" />
+
+        <compatibility:outbound-properties-to-var />
+
+        <compatibility:outbound-properties-to-var>
+            <!--Migration WARN: Instead of using outbound properties in the flow, move the expression that sets the property into the XML attribute (such as 'method') of the operation or listener that accepts the expression.-->
+            <!--    For more information refer to:-->
+            <!--        * https://github.com/mulesoft/mule-migration-assistant/blob/master/docs/user-docs/migration-tool-post-mig.adoc#outbound_properties-->
+        </compatibility:outbound-properties-to-var>
+
+    </flow>
+
+    <sub-flow name="listenerContentType_Enricher_0">
+        <ee:transform doc:name="CALENDARS-S-T-0004">
+            <ee:message>
+                <ee:set-payload><![CDATA[%dw 2.0
+output application/java  
+---
+{
+  message: "Not able to retrieve calendars." as String,
+  status: "500" as String,
+  code: "CALENDARS-S-T-0004" as String,
+  origin: "database" as String,
+  causeMessage: "" as String
+}]]></ee:set-payload>
+            </ee:message>
+        </ee:transform>
+    </sub-flow>
+
+</mule>

--- a/mule-migration-tool-engine/src/main/java/com/mulesoft/tools/migration/report/DefaultMigrationReport.java
+++ b/mule-migration-tool-engine/src/main/java/com/mulesoft/tools/migration/report/DefaultMigrationReport.java
@@ -150,13 +150,13 @@ public class DefaultMigrationReport implements MigrationReport<ReportEntryModel>
           elementToComment.addContent(i++, new Comment("    For more information refer to:"));
 
           for (String link : documentationLinks) {
-            elementToComment.addContent(i++, new Comment("        * " + link));
+            elementToComment.addContent(i++, new Comment("        * " + sanitize(link)));
           }
         }
 
         if (element != elementToComment) {
           XmlDslUtils.removeNestedComments(element);
-          elementToComment.addContent(i, new Comment(outp.outputString(element)));
+          elementToComment.addContent(i, new Comment(sanitize(outp.outputString(element))));
         }
       }
     }

